modules/services/waybar.nix

```diff
@@ -0,0 +1,164 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.waybar;
+
+  configText = builtins.toJSON ({ inherit (cfg) order; } // cfg.extraConfig);
+
+  configFile = pkgs.writeText "config" configText;
+
+in {
+  options = {
+    services.waybar = {
+      enable = mkEnableOption ''
```

Comment:
The description should be adjusted to match the definition of [`mkEnableOption`](https://github.com/NixOS/nixpkgs/blob/2e5672920ed57c5418e6f039072342be7101bf8c/lib/options.nix#L68) so that the description ends up being a correct sentence.